deliveries/Archisinal-milestone_1.md

```diff
@@ -0,0 +1,41 @@
+| 0e. | Article | [Medium: Archisinal Technologies — Genesis Scoping & MVP](https://medium.com/p/3df888e1de98)                                                                            |                                                                                                                                                                                              |
+| 1. | NFT Minting | [ArchNFT](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/arch_nft)        | Implemented in ArchNFT contract (PSP34 logic from OpenBrush). Also ArchNFT contains different metadata such as collection royalty.                                                                                                                                                                                             |
+| 2. | NFT Transfer | [ArchNFT](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/arch_nft)        | Implemented in ArchNFT contract (PSP34 logic from OpenBrush). Also ArchNFT contains different metadata such as collection royalty.                                                           |
+| 3. | Creator’s Administering | [Creator](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/creator)         | Creator contract inherits logic of user contract and adding opportunity to deploy collections.                                                                                               |
+| 4. | Buyer’s Administering | [User](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/user)               | User contract is used to manage user data on-chain, such as NFT-avatars or nicks. Will be extended in the future releases.                                                                   |
+| 5. | Marketplace Administration | [Marketplace](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/marketplace) | Contract that contains auction and just marketplace logic.                                                                                                                                   |
+| 6. | Auction | [Marketplace](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/marketplace) | Auction is a part of `marketplace` contract.                                                                                                                                                 |
+| 7. | Test Network Deployment | [DEPLOYMENTS.md](https://github.com/archisinal/marketplace-contracts/blob/main/DEPLOYMENTS.md)     | Successfully deployed our contracts to Shibuya testing network, contract addresses are provided in `DEPLOYMENTS.md` file.                                                                    |
```

Comment:
@sourabhniyogi maybe you can help me here. When I put these addresses into the Polkaholic website I keep getting [an "unknown network" error](https://polkaholic.io/wasmcontract/0x5d409968325b0e615c6fb7b037b71d6f1fbca563733b4320a6fbcec46d3dde96/30000).

Do you know what's wrong?

Reply:
In the current milestone we deployed contracts on Shibuya. I presume Polkaholic doesn't have this chain in the list. In M2 we'll deploy contracts on Shiden. You can check deployed contracts on shibuya subscan. 

PSP22: https://shibuya.subscan.io/account/VxUmm7bdz1xNKvySnM7qceiP22gjWzXwWjiWGdS57pAtDHu

ArchNFT: https://shibuya.subscan.io/account/YPnpU2YEzFfocNkVT11dsXHCQPHfD2f4An7GkbaZKPRXSnp

User: https://shibuya.subscan.io/account/WmPq43dYcHyDRFs9UGXvLXyiHLHpYhaiN3dWnLnZBh3cuv1

Marketplace: https://shibuya.subscan.io/account/bPX5o2Wyd2tDT3dqgMPzEY2bVFTh8s3weabVJFrHam8wjbq

AccountManager: https://shibuya.subscan.io/account/Y3ZwGe9jeMiyjFUPJVgEGxYfkhKt6f1eMR7RS96PJBq4wxY

Reply:
They [do have Shibuya support](https://polkaholic.io/blocks/shibuya), so this _should_ work imo.

Thanks for sending the links over. Can you add those to the submission document, maybe under a new section called `Deployments` or something like that?